### ISSUE TITLE
BioMed Central Collection - 3 recently accessioned items have no files

### ISSUE THREAD

**Issue by mello99** [VTUL/vtechworks#107]
Hi all, 3 items that were recently exported via SWORD into the BioMed Central Collection (11/4/2015) don't have files associated with them. Their handles are the following:

http://hdl.handle.net/10919/63917
http://hdl.handle.net/10919/63916
http://hdl.handle.net/10919/63915

I can go ahead and find the files on BMC, download them, and then upload them to VTW. I just wanted to mention this in case there's an issue on our end that prevented those files from being deposited into VTechWorks. Thanks so much!

**Comment by keithgee** [comment]
It looks like at least two of these are duplicates, as well. I'm not sure if BioMed Central is trying to send metadata updates, or what.

**Comment by amandafrench** [comment]
I think this and #109 are both fixed, right, @keithgee?

**Comment by keithgee** [comment]
@amandafrench, not yet. _Sorry_, @mello99 I had let slip these problems with BioMed Central for so long.

**Comment by keithgee** [comment]
**TL;DR - These items appear to have been created accidentally during a bulk metadata spreadsheet upload. I've fixed it so the handles redirect to the original items, with files.**

I can't quite pin down what happened, but I don't believe these three DSpace items were created with SWORD ingests. Normally, there would be a sword package ZIP file in the bitstreams section, because it also contains the metadata.

My hypothesis is that these three items were accidentally created by the bulk metadata change/import feature in VTechWorks.

Accession date for the three items is 2015-11-04T19:33ish.
I think that taking timezones into account, this corresponds approximately to these log lines:

**CSV Uploaded to VTechWorks**

```
2015-11-04 14:33:41,542 INFO  org.dspace.app.xmlui.aspect.administrative.FlowMetadataImportUtils @ mlohrey@vt.edu:session_id=F261D5897CD9F1FA2BC45845EC224ED1:ip_addr=128.173.XXX.XX:metadataimport:loading file
```

**Items created - item id 74092 is shown - it corresponds to http://hdl.handle.net/10919/63915**

```
2015-11-04 14:33:56,585 INFO  org.dspace.content.Item @ mlohrey@vt.edu:session_id=F261D5897CD9F1FA2BC45845EC224ED1:ip_addr=128.173.XXX.XX:create_item:item_id=74092 
```

Two of these items, 10919/63915 and 10919/63917, have duplicates in the Biomed Central collection, giving additional credence to the hypothesis. However, I can't find an "original" item for 10919/63916 and am confused.

I withdrew the two duplicated items, 10919/63915 and 10919/63917,through the web interface. Then I used SQL to update the handle database so that the original items (with files) are loaded when visiting the urls of the withdrawn duplicates:

```
update handle set resource_id=26517 where handle_id=63915;
update handle set resource_id=66721 where handle_id=63917;
```

I acquired an open copy of "Knowledge-fused differential dependency network models for detecting significant rewiring in biological networks" from BMC and added it to 63916, the item for which the "original" in VTechWorks is hiding.

:fist: :exclamation: 
Closing!
